src/App.scss

```diff
@@ -42,7 +42,7 @@
   height: 100%;
 }
 
-$wrapper-width: 910px;
+$wrapper-width: 955px;
```

Comment:
Increased the width because the width for the `TermsAndConditions` component width would not fit in the left column when 910px.